packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.js

```diff
@@ -174,6 +174,12 @@ export default class QuanticDateFacet extends LightningElement {
     return '';
   }
 
+  fixDateForAllBrowsers(dateString) {
```

Comment:
The function name doesn't reflect its purpose. Something like `fromSearchApiDate(dateString)` might be easier to understand.

Also, I suggest to make it a utility function we can reuse.